content/en/apps/reference/app-settings/sms.md

```diff
@@ -8,14 +8,15 @@ relevantLinks: >
   docs/apps/concepts/workflows
   docs/design/apps
   docs/apps/guides/africas-talking
+  docs/apps/guides/rapidpro
 keywords: workflows sms
 ---
 
 SMS settings are defined under the `sms` key, as an object supporting the following properties:
 ## `app_settings.json .sms`
 | property         | default       | description                                                                                                                                                                              |
 |------------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| outgoing_service | medic-gateway | Defines the service to use to send SMS messages. Currently supports "medic-gateway" and "africas-talking". Read more about "africas-talking" configuration [here]({{% ref "apps/guides/messaging/africas-talking" %}}). |
+| outgoing_service | medic-gateway | Defines the service to use to send SMS messages. Currently supports "medic-gateway", "africas-talking" or "rapidpro". Read more about "africas-talking" configuration [here]({{% ref "apps/guides/messaging/africas-talking" %}}). Read more about "rapidpro" configuration [here]({{% ref "apps/guides/messaging/rapidpro" %}}). |
```

Comment:
I try to avoid the "click here" style of link. What about...

```suggestion
| outgoing_service | medic-gateway | Defines the service to use to send SMS messages. Currently supports "medic-gateway", "africas-talking" or "rapidpro". For more information read the documentation on ["africas-talking" configuration]({{% ref "apps/guides/messaging/africas-talking" %}}) and ["rapidpro" configuration]({{% ref "apps/guides/messaging/rapidpro" %}}). |
```

Reply:
Done